version: '3'
services:

  nginx:
    image: nginx:1.15-alpine
    ports:
      - "80:80"
      - "443:443"
    volumes:
      - ./data/nginx:/etc/nginx/conf.d
      - ./data/certbot/conf:/etc/letsencrypt
      - ./data/certbot/www:/var/www/certbot
    restart_policy: 
      condition: on-failure
 
  certbot:
    image: certbot/certbot
    volumes:
      - ./data/certbot/conf:/etc/letsencrypt
      - ./data/certbot/www:/var/www/certbot
    restart_policy:
      condition: on-failure
      
   portainer:
     image: portainer/portainer
     ports:
       - "9000:9000"
     volumes:
       - ./var/run/docker.sock:/var/run/docker.sock
     restart_policy:
       condition: on-failure
       
    watchtower:
      image: v2tec/watchtower
      volumes:
        - /var/run/docker.sock:/var/run/docker.sock
     restart_policy:
       condition: on-failure
       
